python/pyiceberg/cli/output.py

```diff
@@ -134,6 +140,9 @@ def uuid(self, uuid: Optional[UUID]):
 class JsonOutput(Output):
     """Writes json to stdout"""
 
+    def __init__(self, **properties: Any):
```

Comment:
We could also move this to the constructor of the parent `Output` class, and then we don't have to implement it at both the `JsonOutput` and `ConsoleOutput`.

Reply:
I was wondering if we should do that since the parent is an abstract base class. Is that a good idea?